internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -17,8 +17,40 @@ CloudFrontDistribution:
             OriginProtocolPolicy: match-viewer # Will need to change with TLS termination and Alias usage
           DomainName: !GetAtt PublicLoadBalancer.DNSName
           Id: !Sub 'copilot-${AppName}-${EnvironmentName}-origin'
-      # ViewerCertificate: # Used for Aliases
-      #   AcmCertificateArn: # Imported certificate goes here
-      #   CloudFrontDefaultCertificate: true # If importing certificate, set to false
-      #   MinimumProtocolVersion: TLSv1 # Default for now, may need to change later
-      #   SsLSupportMethod: sni-only # Changing this can incur costs to the user
+{{- if not .PublicHTTPConfig.ImportedCertARNs}}
```

Comment:
In a follow up PR, this will also take the `cdn.cert` field in manifest into consideration, is this understanding correct?

Reply:
I imagine there will be an `{{- else}}` case at the end of this to cover the viewer certificate for when the user imports certificates

Reply:
Yeah it will be the next pr